Avalonia.MusicStore/ViewLocator.cs

```diff
@@ -7,9 +7,9 @@ namespace Avalonia.MusicStore
 {
     public class ViewLocator : IDataTemplate
     {
-        public IControl Build(object data)
+        public Control Build(object? data)
         {
-            var name = data.GetType().FullName!.Replace("ViewModel", "View");
+            var name = data!.GetType().FullName!.Replace("ViewModel", "View");
```

Comment:
Ah, nevermind, if we use reflection here, we still need that Roots.xml file. 
I guess we shouldn't remove this view locator, if it's used in the tutorials, without updating tutorials first.

Reply:
Yeah in long term I want to replace MusicStore by somehting more useful.